terminus-ui/sort/src/sort-header.component.html

```diff
@@ -7,6 +7,7 @@
     type="button"
     [attr.aria-label]="_intl.sortButtonLabel(id)"
     [attr.disabled]="_isDisabled() || null"
+    (click)="_handleClick()"
```

Comment:
curious why you have `_` here?

Reply:
Legacy material code. When sort was originally pulled into the library I didn't have time to fully refactor etc.

There is a `FIXME` note at the top of the file to refactor out the dangles and remove the lint rule for it... someday....